libnmstate/nm/applier.py

```diff
@@ -458,10 +459,12 @@ def _build_connection_profile(
     if base_profile:
         con_setting.import_by_profile(base_con_profile)
         con_setting.set_profile_name(iface_name)
+        lldp.apply_lldp_setting(con_setting, iface_desired_state)
     else:
         con_setting.create(
             con_name=iface_name, iface_name=iface_name, iface_type=iface_type,
         )
+        lldp.apply_lldp_setting(con_setting, iface_desired_state)
```

Comment:
Two duplicated lines is not needed, single one with upper indent would works.

Reply:
You are right, thanks! Done!